app/forms/silo-create.tsx

```diff
@@ -36,6 +36,11 @@ const defaultValues: SiloCreateFormValues = {
   tlsCertificates: [],
   siloAdminGetsFleetAdmin: false,
   siloViewerGetsFleetViewer: false,
+  quotas: {
+    cpus: 0,
+    memory: 0,
+    storage: 0,
+  },
```

Comment:
So this is a good start _but_ we want to add the form elements where the operator can actually set these values. That can be done in a separate PR if you wish though.